test/apify/crawlers/puppeteer_crawler.test.ts

```diff
@@ -30,6 +30,8 @@ describe('PuppeteerCrawler', () => {
     let requestList: RequestList;
     let servers: ProxyChainServer[];
     let target: Server;
+    let serverUrl : string;
+    let proxyConfiguration : ProxyConfiguration;
```

Comment:
how come this is not caught by eslint

```suggestion
    let serverUrl: string;
    let proxyConfiguration: ProxyConfiguration;
```